api/STYLE.md

```diff
@@ -122,8 +122,7 @@ To add an extension config to the API, the steps below should be followed:
        deps = ["@com_github_cncf_udpa//udpa/annotations:pkg"],
    )
    ```
-1. Add to the v3 extension config proto `import "udpa/annotations/migrate.proto";`
-   and `import "udpa/annotations/status.proto";`
+1. Add to the v3 extension config proto `import "udpa/annotations/status.proto";`
```

Comment:
not sure about this one - this is also added and unused frequently - but im guessing it is needed (in some cases ?) and may relate to some of the docs below ?

Reply:
This is needed for the `work_in_progress` annotation below.

Reply:
k - on that basis lets keep - i think the issue is more status not being set than the status proto being included